### ISSUE TITLE
Theme Pack Utils bundle not starting

### ISSUE THREAD

**Issue by jgubo-mct** [anozimada/idempiere-themepack#3]
I installed both, the fragment and utils bundle using the Felix Console.
But the Theme Pack Utils is stuck in Status "Starting" on the console, no errors at all in the idempiere logs.
I also can't uninstall the bundle, just the fragment. Refresh of Packages doesn't work either. Still just "Starting".

iDempiere Version 9.0.0.202205181744

Any help is greatly appreciated! Thanks!

Regards Johannes

**Comment by anozimada** [comment]
I just installed com.anozimada.idempiere.themepack.utils_1.1.0.jar on https://demo.globalqss.com/ (9.0.0.202205181744) and https://test.idempiere.org/ (10.0.0.202206080130) and it's running just fine, the status is "Active".

1. Did you follow the instruction precisely https://wiki.idempiere.org/en/Developing_Plug-Ins_-_Get_your_Plug-In_running#Apache_Felix_Web_Console ?
2. Can you try install other plugin and see if it works?
3. In Apache Felix Web Console, can you try clicking the plugin to see the plugin details and see if there's any hint?

**Comment by jgubo-mct** [comment]
That's interesting. Yes I followed all the instructions. But now that you mention it, there are a couple more plugins/bundles in the state "Starting", could be a problem that at some point started and never been recognized. The setup has been updated from 7 to 9 over the past years. Upgrading iDempiere to a new major version is still awful.

If I find a fix for the root cause, I'll let you know here.

**Comment by jgubo-mct** [comment]
Okay, it's working now. I first had to remove the osgi package stuck in "Starting" manually from the server, since there is no Remove Button in the Felix Web Console (Path: ), Refresh Packages, Restart iDempiere Service.

Then I installed utils first with the following settings and then the theme fragment. It works :) Thanks for your contribution!

![image](https://user-images.githubusercontent.com/52563802/172588414-02ed43fc-c4dd-437c-855d-c9a906c41fdd.png)
